src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2320,6 +2320,9 @@ bool Creature::LoadCreaturesAddon()
         }
     }
 
+    if (uint32 visibilityRange = cainfo->visibilityRange)
```

Comment:
No, it's declaring a variable inside the if and assigning it the value of `cainfo->visibilityRange`